deploy.sh

```diff
@@ -25,6 +24,10 @@ for i in $(seq 0 $(($NUM_SYMBOLS - 1))); do ./deploy-symbols.sh $i; done
 for i in $(seq 0 $(($NUM_INDEXED_SEARCH - 1))); do ./deploy-zoekt-indexserver.sh $i; done
 for i in $(seq 0 $(($NUM_INDEXED_SEARCH - 1))); do ./deploy-zoekt-webserver.sh $i; done
 
+# Postgres/codeintel-db must be started before these.
+./deploy-precise-code-intel-bundle-manager.sh
+./deploy-precise-code-intel-worker.sh
```

Comment:
Please undo this change / move these back to their original position. The note below talking about Redis is describing something about file permissions (which I understand is a bit odd.)

The services themselves need to be able to start in any order, allowing for restarts if they depend on another service, and in that respect precise-code-intel should be no different than frontend. e.g. we cannot enforce startup order in Docker Compose.

Or am I missing something here?

Reply:
Undone.